<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>628</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Матричные игры</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="MatrInitLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="ConfirmParamsButton">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>16</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Подтвердить параметры</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="MatrixParamsHLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="MatrixParamsVLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="PlayerNameLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="PlayerLabel">
              <property name="font">
               <font>
                <family>Arial</family>
               </font>
              </property>
              <property name="text">
               <string>Имена Игроков:     </string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Name1Edit"/>
            </item>
            <item>
             <widget class="QLineEdit" name="Name2Edit"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="SizeLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>10</number>
            </property>
            <item>
             <widget class="QLabel" name="OptionLabel">
              <property name="font">
               <font>
                <family>Arial</family>
               </font>
              </property>
              <property name="text">
               <string>Количество Опций:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Option1Edit">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
	border-color: rgb(255, 25, 55);
	border-width: 10px;
}</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Option2Edit"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QRadioButton" name="BiMatrixButton">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Биматричная</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="ConfirmMatrixButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Подтвердить матрицу</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="FunctionHLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QComboBox" name="PlayerChoiceBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="BufferButton">
          <property name="text">
           <string>Посмотреть результат</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="MatrixInputLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuAlgorithms">
    <property name="title">
     <string>Алгоритмы</string>
    </property>
    <addaction name="actionMinMax"/>
    <addaction name="actionMaxMin"/>
    <addaction name="actionDominant"/>
    <addaction name="actionWeaklyDominant"/>
    <addaction name="actionNesh"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAlgorithms"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionMinMax">
   <property name="text">
    <string>МинМакс</string>
   </property>
  </action>
  <action name="actionMaxMin">
   <property name="text">
    <string>МаксМин</string>
   </property>
  </action>
  <action name="actionDominant">
   <property name="text">
    <string>Строго доминируемая</string>
   </property>
  </action>
  <action name="actionWeaklyDominant">
   <property name="text">
    <string>Слабо доминируемая</string>
   </property>
  </action>
  <action name="actionNesh">
   <property name="text">
    <string>Равновесие по Нэшу</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
